fdbserver/EncryptKeyProxy.actor.cpp

```diff
@@ -366,14 +427,23 @@ ACTOR Future<Void> getLatestCipherKeys(Reference<EncryptKeyProxyData> ekpProxyDa
 	for (const auto& info : dedupedDomainInfos) {
 		const auto itr = ekpProxyData->baseCipherDomainIdCache.find(info.first);
 		if (itr != ekpProxyData->baseCipherDomainIdCache.end() && itr->second.isValid()) {
-			cachedCipherDetails.emplace_back(info.first, itr->second.baseCipherId, itr->second.baseCipherKey, arena);
+			ASSERT(!itr->second.isExpired());
```

Comment:
should handle this case (e.g. ignore the cached key) instead of assert, since in the extreme case if the refresh didn't run in time, a cached key could be expired?

Reply:
Agreed. 
Even though above line (line 429) checks CipherKey validity, it is possible that `now()` increment would make a key expire. 
Thanks for pointing, will fix it.